apps/frontend/src/libs/components/user-popup/user-popup.tsx

```diff
@@ -0,0 +1,34 @@
+import { useAppDispatch, useCallback } from "~/libs/hooks/hooks.js";
+import { actions as authActions } from "~/modules/auth/auth.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	email: string;
+	name: string;
+};
+
+const UserPopup = ({ email, name }: Properties): JSX.Element => {
+	const dispatch = useAppDispatch();
+
+	const handleLogout = useCallback((): void => {
+		void dispatch(authActions.logout());
+	}, [dispatch]);
+
+	return (
+		<div className={styles["user-popup"]}>
+			<div className={styles["user-info"]}>
+				<p className={styles["user-name"]}>{name}</p>
+				<p className={styles["user-email"]}>{email}</p>
+			</div>
+			<div className={styles["buttons"]}>
+				<button className={styles["button"]}>Profile</button>
+				<button className={styles["button"]} onClick={handleLogout}>
+					Log out
+				</button>
```

Comment:
Shouldn't we use the Button component?

Reply:
The button here needs different styling and onClick prop. So Button component is not very suitable for that.